src/Component/CodeEditor/CodeEditor.tsx

```diff
@@ -275,6 +281,18 @@ export const CodeEditor: React.FC<CodeEditorProps> = ({
   });
   parserOptions = [...parserOptions, ...additionalOptions];
 
+  const onFileChanged = (info: UploadChangeParam) => {
+    const reader = new FileReader();
+    reader.onload = e => {
+      const str = reader.result as string;
+      onChange(str);
+    };
+    let l = info.fileList.length;
+    if (l === 1) {
+      reader.readAsText(info.fileList[l-1].originFileObj);
+    }
```

Comment:
```suggestion
      reader.readAsText(info.fileList[0].originFileObj);
```
As you specify `maxCount={1}` below you can assume that there is always juste one file in the fileList.